angular/src/directives/navigation/router-link-delegate.ts

```diff
@@ -31,19 +37,15 @@ export class RouterLinkDelegateDirective implements OnInit, OnChanges {
     this.updateTargetUrlAndHref();
   }
 
+  @HostListener('click')
```

Comment:
Do we still need the "@internal" comment?

Reply:
Apologies to the user I just tagged by accident 😂

Reply:
I don't believe so. I'm unsure why it was marked as `@internal` in the first place. Not a convention that exists in the Angular code base for a similar implementation: https://github.com/angular/angular/blob/master/packages/router/src/directives/router_link.ts#L238-L239

Also not a method that developers are accessing through a public API.